client/verta/tests/test_config.py

```diff
@@ -147,6 +150,12 @@ def test_merge_and_overwrite(self, expected_config):
         with _config_utils.read_merged_config() as config:
             assert config == expected_config
 
+    def test_skip_client_load_config(self, expected_config):
+        assert Client()._config == expected_config
+
+        os.environ["VERTA_DISABLE_CLIENT_CONFIG"] = "1"
+        assert Client()._config == dict()
+
```

Comment:
I tried writing a test for `get_possible_config_file_dirs()` skipping inaccessible directories, but I hit the time cap that I set for myself. Since the environment variable has coverage here, and I manually tested inaccessible directories (see PR description), I'd like to ship this and come back to that other test afterwards.